lib/util/systimemon/systime_mon.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package systimemon
+
+import (
+	"time"
+
+	"github.com/pingcap/tidb/util/logutil"
+	"go.uber.org/zap"
+)
+
+// StartMonitor calls systimeErrHandler if system time jump backward.
+func StartMonitor(now func() time.Time, systimeErrHandler func(), successCallback func()) {
```

Comment:
Should we add `context` to cancel it? Though it is a lifelong task. But tests may fail if we add go routine leak checker later.